domains/measurements/Measurements.API/Tests/Fixtures/TestServerHelpers/ForwardLoggingProvider.cs

```diff
@@ -0,0 +1,74 @@
+#region Copyright notice and license
+// Copyright 2019 The gRPC Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/grpc/test-services/sample/Tests/Server/IntegrationTests/Helpers/ForwardingLoggerProvider.cs
+#endregion
+
+using System;
+using Microsoft.Extensions.Logging;
+
+namespace Tests.Fixtures.TestServerHelpers
+{
+    internal class ForwardingLoggerProvider : ILoggerProvider
```

Comment:
Getting a warning regarding IDisposable. I am also unsure what this class actual does